core/contractHost.js

```diff
@@ -114,9 +118,19 @@ No invites left`;
               },
               redeem,
             });
-            return contract(terms, inviteMaker);
+            return contract.start(terms, inviteMaker);
```

Comment:
It seems like there are now three methods to call at the beginning: contractHost.install, contractHost.spawn (and spawn calls contract.start). I think adding `start` is a good choice, because contract() seems like a noun acting like a verb, but we might want extra documentation on what makes these three different.

Reply:
Yeah, I'm working on documentation now. I think the fact that this change forced us to migrate the contracts from a pure function to a thunk is pretty unfortunate. I'm going to see if we can find a way to have the contractHost take in a function and do the extra wrapping that was required.

Requiring contract designers to write 

> const coveredCallThunk = () =>
>   const [	  harden({
>     escrowExchangeInstallationP,	    start(terms, inviteMaker) {

instead of the old 

> function coveredCall(terms, inviteMaker) {

seems like a mistake.

Reply:
The new version is back to 

> function coveredCall(terms, inviteMaker) {

But the end has changed from exporting the function to exporting a record containing labelled strings for the function sources.

> const coveredCallSrc = {
>  start: `${coveredCall.start}`,
>  checkAmount: `${coveredCall.checkAmount}`,
> };
>
> export { coveredCallSrc };